From 8365487a0cfec2aa1364c48137158822acf77181 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Wed, 4 Feb 2015 16:20:33 +0800
Subject: [PATCH 0097/5242] MLK-10202-2 doc: usb: mxs-phy: add tx-d-cal
 property

commit  5468fd884015eb54616231cc5984f8b6810a0c7e from
https://source.codeaurora.org/external/imx/linux-imx.git

Add tx-d-cal property

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit ebb97b3d3ca99516d3daac09813ae480de2adffe)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/phy/mxs-usb-phy.txt        |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
index 6ac98b3..21aad8e 100644
--- a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
+++ b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
@@ -21,6 +21,9 @@ Optional properties:
   that terminates the DP output signal. Default: 45
 - fsl,tx-d-cal: Integer [79-119]. Current trimming value (as a percentage) of
   the 17.78mA TX reference current. Default: 100
+- tx-d-cal: Try to adjust this value to improve signal quality, and pass
+  USB Certification, the value is from 0x0 to 0xf, and the register offset
+  is 0x10 (USBPHY_TX).
 
 Example:
 usbphy1: usbphy@20c9000 {
@@ -28,4 +31,5 @@ usbphy1: usbphy@20c9000 {
 	reg = <0x020c9000 0x1000>;
 	interrupts = <0 44 0x04>;
 	fsl,anatop = <&anatop>;
+	tx-d-cal = <0x5>;
 };
-- 
1.7.9.5

